styles/styles.css

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  width: 310px;
+  height: 50px;
+  margin-top: 120px;
+  margin-right: auto;
+  margin-left: auto;
+  text-align: center;
+  font-size: 40px;
+}
+
+.highlight {
+  margin: auto;
+  width: 100px;
+  height: 5px;
+}
+
+.info {
+  display: inline-block;
+  width: 25%;
+  height: 180px;
+  float: left;
```

Comment:
Look at the cases when you could use this attribute:
https://developer.mozilla.org/en-US/docs/Web/CSS/float

The reason of drop out the last element is margin of body element and width 100% of the offer div.